--- ansiblelaunchserver.py	2017-10-26 12:47:17.866793689 +0000
+++ ansiblelaunchserver.py.new	2017-10-26 12:47:53.359823511 +0000
@@ -1511,6 +1511,7 @@
         with open(fn, 'w') as config:
             config.write('[defaults]\n')
             config.write('hostfile = %s\n' % jobdir.inventory)
+            config.write('inventory = %s\n' % jobdir.inventory)
             config.write('local_tmp = %s/.ansible/local_tmp\n' % jobdir.root)
             config.write('remote_tmp = %s/.ansible/remote_tmp\n' % jobdir.root)
             config.write('private_key_file = %s\n' % self.private_key_file)
